extensions/interactions/GraphInput/directives/GraphInput.js

```diff
@@ -45,49 +48,58 @@ oppia.directive('oppiaInteractiveGraphInput', [
             isWeighted: false,
             isLabeled: false
           };
-
           $scope.submitGraph = function() {
             // Here, angular.copy is needed to strip $$hashkey from the graph.
             $scope.onSubmit({
               answer: angular.copy($scope.graph),
               rulesService: graphInputRulesService
             });
           };
+          $scope.interactionIsActive = ($scope.getLastAnswer() === null) ||
+            ($scope.getLastAnswer() === undefined);
```

Comment:
Ditto, the undefined case shouldn't happen.

Reply:
Done, only null case is being checked now.